doozerlib/dblib.py

```diff
@@ -470,6 +475,14 @@ def __init__(self, db, table, extras={}, dry_run=False):
 
         self.attrs.update(extras)
 
+        # An allow list for labels that start with io.openshift
+        self.io_label_allow_list = [
+            'label_io_openshift_build_source_location',
+            'label_io_openshift_build_commit_id',
+            'label_io_openshift_build_commit_url',
+            'label_io_openshift_release_operator',
+            'label_io_openshift_build_versions']
```

Comment:
yeah... don't really want this here to affect every future use of `Record`....

Reply:
Changed to distgit.py